docs/install/install-local-new.md

```diff
@@ -0,0 +1,300 @@
+* Step 9: Test the Site
+* Step 10: Ingest Sample Objects
+* Step 11: Check-in the newly created Drupal / Islandora site code into a git repository
+
+---
+
+## Step 1: Edit `/etc/hosts` File
+
+Enable the Local ISLE Installation to be viewed locally on workstation browser as: e.g. `https://yourprojectnamehere.localdomain`.
+
+* Please use these instructions to [Edit the "/etc/hosts" File](../install/install-Local-edit-hosts-file.md).
+
+* Replace `isle` or `yourprojectnamehere` with your domain / project name e.g. `yourprojectnamehere.localdomain`
+
+---
+
+## Step 2: Setup git for the ISLE project
+
+**Please note:** The commands given below are for command line usage of git. GUI based clients such as the [SourceTree App](https://www.sourcetreeapp.com/) may be easier for endusers to use for the git process.
+
+* Within your git repository provider / hoster e.g [Github](https://github.com), [Bitbucket](https://bitbucket.org/), [Gitlab](https://gitlab.com), create two new empty git repositories:
+  * ISLE project config - e.g. `yourprojectnamehere-isle`
+    * This Git repository will hold your copy of the ISLE code along with your environment-specific customizations. Storing this in a code repository and following the workflow described here will save you a lot of time and confusion.
+  * Drupal / Islandora site code - e.g. `yourprojectnamehere-islandora`
+    * This Git repository will hold your copy of the Drupal / Islandora code along with your site specific customizations. Storing this in a code repository and following the workflow described here will save you a lot of time and confusion.
+
+The git project name can be your institution name or the name of the collections you plan to deploy; your choice entirely. A very clear distinction between the ISLE and Islandora code should be made in the repo name. Do not confuse or label Drupal / Islandora site code as ISLE and vice-versa.
+
+* Clone this newly created ISLE project to your local laptop or workstation.
+  * Open a `terminal` (Windows: open `PowerShell`)
+  * Navigate to an directory that will house your new ISLE project directory using the `cd` command.
+  * `git clone https://yourgitproviderhere.com/yourinstitutionhere/yourprojectnamehere-isle.git`
+
+* Navigate to this directory
+  * `cd /path/to/your/repo`
+
+* Add the ICG ISLE git repository as a git upstream. (_the code below is a real path and command_)
+  * `git remote add icg-upstream https://github.com/Islandora-Collaboration-Group/ISLE.git`
+  * you can check by running this command: `git remote -v`
+
+```bash  
+icg-upstream	https://github.com/Islandora-Collaboration-Group/ISLE.git (fetch)
+icg-upstream	https://github.com/Islandora-Collaboration-Group/ISLE.git (push)
+origin	https://yourgitproviderhere.com/yourinstitutionhere/yourprojectnamehere-isle.git (fetch)
+origin	https://yourgitproviderhere.com/yourinstitutionhere/yourprojectnamehere-isle.git (push)
+```
+
+* Run a git fetch
+  * `git fetch icg-upstream`
```

Comment:
This is unnecessary as the `git pull` does both a fetch and merge.  Do you have reason you like to do it this way?  Thanks!

Reply:
Yeah it is probably overkill but I've found in the past that I've not always gotten the "latest" updates from running `git pull` so I've started to run `git fetch` as a pre-caution. I tend to jump between using git cli and the Sourcetree Mac app. I don't know which process fails me but one isn't consistent in getting branch updates from the remote(s).